### PR TITLE
Add nuimages gif

### PR DESCRIPTION
**Pull request by xiliu8006**
add a nuimages demo in configs/nuimages/README.md

### CODE REVIEW

**Review by Tai-Wang — APPROVED** [review]
Good demo, but is it convenient to use some brightly colored/colorful pictures :)

File: configs/nuimages/README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ The convert script can be found [here](https://github.com/open-mmlab/mmdetection
 The baseline results include instance segmentation models, e.g., Mask R-CNN, Cascade Mask R-CNN, and HTC.
 We will support panoptic segmentation models in the future.
 
+![demo image](../../resources/nuimages_demo.gif)
```
Comment (ZwwWayne) [inline configs/nuimages/README.md:13]: The gif file is too big, may save as JEPG images first and then cat them as gif.

**Comment by xiliu8006** [comment]
> Good demo, but is it convenient to use some brightly colored/colorful pictures :)

I am sorry, I only have the nuimages v1.0-mini dataset and it is the best one I found

File: configs/nuimages/README.md (hunk 1 above)
Comment (xiliu8006) [reply configs/nuimages/README.md:13]: The images are JEPG format, but I can compress the gif to reduce the size. I will push it soon.

Comment (ZwwWayne) [inline configs/nuimages/README.md:13]: May add HTML code to put the demo in the center and closer to other contents.

**Pull request merged**